.github/workflows/backend-lint-test.yml

```diff
@@ -32,7 +32,12 @@ jobs:
           version: v1.55.2
```

Comment:
Do we still need to install the go binary and linter? These are installed locally as part of the task command right?